src/docs/asciidoc/reference/configuration/account-configuration.adoc

```diff
@@ -1,9 +1,13 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
-To use ZetaPush Celtia, you need to have an account and an optional application.
-You specify this properties in the `.zetarc` file.
-Here is the list of them:
+To use `ZetaPush`, you need to have a _ZetaPush account_ and a _ZetaPush application_.
+If you don't have any of them, you can link:{free-trial-url}[signup for a free trial] or link:{contact-url}[contact us for more suitable account].
+
+Then, you can set your account configuration in the `.zetarc` file.
+This file is present at the root of your application folder.
+
+Here is the list of properties that can be used. For each of them, you also have the associated <<reference#cli, cli flag>> and environment variable. 
```

Comment:
We should indicate priority order: CLI parameter, environment variable and property in .zetarc

We should also indicate that we use cosmiconfig (https://github.com/davidtheclark/cosmiconfig) for zetarc and that file can be placed somewhere else (in home directory for example). Check if this feature is still available

Reply:
The priority order is indicated in the "configuration override" section.
About `cosmiconfig` the behavior is not correct. (https://github.com/zetapush/zetapush/issues/290)
So I don't talk about that for now.